SourcingTests/CDTrain+CoreDataProperties.swift

```diff
@@ -13,6 +13,12 @@ import Foundation
 import CoreData
 
 extension CDTrain {
+    
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<CDTrain> {
+        return NSFetchRequest<CDTrain>(entityName: "CDTrain")
```

Comment:
change this to
```swift
public static func fetchRequest() -> NSFetchRequest<CDTrain> {
       return NSFetchRequest(entityName: "CDTrain")
}

Reply:
then, I get error: 

Method 'fetchRequest()' with Objective-C selector 'fetchRequest' conflicts with method 'fetchRequest()' from superclass 'NSManagedObject' with the same Objective-C selector

Reply:
could be static var instead... but it is ok if we stay with your solution

Reply:
👍